integration/src/test/resources/org/mule/test/integration/locator/component-locator-error-mapping.xml

```diff
@@ -27,11 +33,23 @@
         <logger/>
     </flow>
 
-    <flow name="errorMappingFlowDeclaration">
+    <flow name="errorMappingFlow">
         <http:request method="GET" config-ref="HTTP_Request_configuration">
             <error-mapping targetType="APP:ERROR_TYPE_1"/>
         </http:request>
     </flow>
 
+    <http:request-config name="HTTP_Request_configuration">
```

Comment:
Is this relevant?

Reply:
Yes, as when I send to lazyInit "errorMappingFlow" for the test, it has a dependency to the "HTTP_Request_configuration" therefore it will not be able to start the flow (actually the request)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name '(inner bean)#8a0ab4c': Cannot resolve reference to bean 'HTTP_Request_configuration' while setting bean property 'parameters' with key [config-ref]; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No bean named 'HTTP_Request_configuration' available